addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/SpiderThread.java

```diff
@@ -533,4 +526,58 @@ public void onBrowserCreated(EmbeddedBrowser arg0) {
             // Nothing to do.
         }
     }
+
+    @Getter
+    static class WebDriverProcess {
+
+        private static final String LOCAL_PROXY_IP = "127.0.0.1";
+        private static final int INITIATOR = HttpSender.AJAX_SPIDER_INITIATOR;
```

Comment:
Why store this a second time instead of using it directly?

Reply:
To use a single reference, it's not storing a second time, it will be inlined.